internal/server/templates/index.html

```diff
@@ -11,61 +11,157 @@
 			padding: 0;
 			height: 100vh;
 		}
+		div {
+			box-sizing: border-box;
+		}
+		pre {
+			white-space: pre-wrap;
+			width: 100%;
+		}
+		label {
+			display: block;
+			font-family: Verdana, Arial, Helvetica, sans-serif;
+			font-size: 1.0rem;
+		}
+		input {
+  			padding-bottom: 0.1rem;
+		}
 		.page-container {
 			display: flex;
 			flex-direction: column;
 			gap: 0.6rem;
-			box-sizing: border-box;
 			height: 100%;
 			padding: 0.6rem;
 		}
 		.controls-container {
-			background-color: #fff;
+			width: 100%;
+			background-color: #ffffff;
+			border: 1px solid #414141;
+			padding: 0.6rem;
+		}
+		.controls-container form {
+			display: flex;
+			flex-direction: column;
+		}
+		.controls-container label {
+			padding-bottom: 0.3rem;
+		}
+		.url-entry-container {
+			display: flex;
+			align-items: center;
+			gap: 0.3rem;
+		}
+		.url-entry-container input[type="url"] {
+			flex-grow: 1;
+		}
+		.url-entry-container input[type="submit"] {
+			padding-inline: 0.6rem;
+		}
+		.token-container {
+			background-color: #ffffff;
+			border: 1px solid #414141;
+			padding: 0.6rem;
+			display: flex;
+			flex-direction: column;
+			gap: 0.6rem;
+		}
+		.token-header {
+			display: flex;
+			justify-content: start;
+			align-items: center;
+			gap: 0.3rem;
+		}
+		.token-header a {
+			font-size: 1.0rem;
+			font-family: Verdana, Arial, Helvetica, sans-serif;
+			text-decoration: none;
+			margin-bottom: 1px;
```

Comment:
This `margin-bottom` and the `line-height`s here and in the next class are there because the vertical alignment of the +/- and the text following it were slightly off. There is now 1px extra space below, but the alignment looks ok. 

This is annoying, but not that big of a deal -- but if there's a better way to do this, I'd like to know!